doc: Update versions for 2.2.0 release (#877)
nrfconnect/sdk-nrfxlib: gzll/CHANGELOG.rst

> All notable changes to this project are documented in this file.

Reviewer: ```suggestion
All the notable changes to this project are documented on this page.
```

All notable changes to this project are documented in this file.

nRF Connect SDK v2.2.0
**********************

All the notable changes included in the |NCS| v2.2.0 release are documented in this section.

> * Support nRF5340 network core.

Reviewer 2: Suggest removing this 'support nRF534' line instead. The style is thus more consistent with the other nrfxlib changelogs.

Author: fixed
